content/references/protocol-reference/transactions/transaction-types/nftokenacceptoffer.md

```diff
@@ -40,7 +40,7 @@ In direct mode, `NFTokenAcceptOffer` transaction fails if:
 
 * The `NFTokenOffer` against which `NFTokenAcceptOffer` transaction is placed is an offer to `buy` the `NFToken` and the account executing the `NFTokenAcceptOffer` is not, at the time of execution, the current owner of the corresponding `NFToken`.
 * The `NFTokenOffer` against which `NFTokenAcceptOffer` transaction is placed is an offer to `sell` the `NFToken` and was placed by an account which is not, at the time of execution, the current owner of the `NFToken`.
-* The `NFTokenOffer` against which `NFTokenAcceptOffer` transaction is placed is an offer to `sell` the `NFToken` and was placed by an account which is not, at the time of execution, the `Account` in the recipient field of the `NFTokenOffer`, if one exists.
+* The `NFTokenOffer` against which `NFTokenAcceptOffer` transaction is placed is an offer to `sell` the `NFToken` and was placed by an account which is not, at the time of execution, the `Destination` in the recipient field of the `NFTokenOffer`, if one exists.
```

Comment:
Good catch, I think this needs a little more of a rewrite (I had a hell of a time trying to parse the meaning, here).

"The account initiating a `NFTokenAcceptOffer` transaction against a `NFTokenOffer` to sell a `NFToken` is different than an account identified in the optional `Destination` field of the `NFTokenOffer`."

Reply:
There is no "recipient" field. Please reconsider my recommended change.